x-pack/test/functional/es_archives/endpoint/resolver/signals/mappings.json

```diff
@@ -3,7 +3,6 @@
   "value": {
     "aliases": {
       ".siem-signals-default": {
-        "is_write_index": true
```

Comment:
Can't set this as a write index because there is already an index installed that is the write index.